File: vm_Assignments/vmc/L2.7/prime.c
#include<stdbool.h>
#include<stdio.h>
#include<math.h>
bool is_divisible(int a, int b)
{
    if(a%b ==0)
        return true;
    else
    {
        return false;
    }
}

bool is_prime(int number)
{
    if(number>1)
    {
        for(int i=2; i<number; i++)
        {
            if(is_divisible(number, i))
                return false;
            else
            {
                return true;
            }
        }
    }
}
int main()
{
    is_prime(4);
    printf("%s", is_divisible(5, 2));
}

File: vm_Assignments/vm/Beginning/L2.7/prime.py
import math
def is_divisible(a, b):
    if a % b == 0:
        return True
    return False


def is_prime(number):
    assert number > 1
    for i in range(2, number):
        if is_divisible(number, i):
            break
    else:
        print(number)

for number in range(2,100):
    is_prime(number)